db-scripts/src/main/resources/cgds.sql

```diff
@@ -802,10 +803,20 @@ CREATE TABLE `reference_genome_gene` (
     FOREIGN KEY (`ENTREZ_GENE_ID`) REFERENCES `gene` (`ENTREZ_GENE_ID`) ON DELETE CASCADE
 );
 
+-- --------------------------------------------------------
+CREATE TABLE `data_access_tokens` (
+    `TOKEN` varchar(50) NOT NULL,
+    `USERNAME` varchar(255) NOT NULL,
```

Comment:
**USERNAME** need to be normalized with **EMAIL** in **user** table to varchar(128).

Reply:
thanks, good catch @khzhu